src/controllers/dynakube/dynakube_controller.go

```diff
@@ -269,6 +285,18 @@ func (controller *Controller) reconcileDynaKube(ctx context.Context, dynakube *d
 	return controller.reconcileComponents(ctx, dynatraceClient, dynakube)
 }
 
+func (controller *Controller) reconcileConnectionInfo(ctx context.Context, dynakube *dynatracev1beta1.DynaKube, dynatraceClient dtclient.Client) error {
+	err := connectioninfo.NewReconciler(ctx, controller.client, controller.apiReader, controller.scheme, dynakube, dynatraceClient).Reconcile()
+	if errors.Is(err, connectioninfo.NoOneAgentCommunicationHostsError) {
+		// missing communication hosts is not an error per se and shall not stop reconciliation, just make sure next reconciliation is happening ASAP
+		// this situation will clear itself after AG has been started
+		controller.setRequeueAfterIfNewIsShorter(errorUpdateInterval)
```

Comment:
> ... is not an error ...

but still using the `errorUpdateInterval` reads a bit backwards 😄 
we could rename `errorUpdateInterval` to something better